site/_docs/howto.md

```diff
@@ -990,6 +992,12 @@ address. You can use
 [the 1.20.0 announcement](https://mail-archives.apache.org/mod_mbox/www-announce/201906.mbox/%3CCA%2BEpF8tcJcZ41rVuwJODJmyRy-qAxZUQm9OxKsoDi07c2SKs_A%40mail.gmail.com%3E)
 as a template. Be sure to include a brief description of the project.
 
+Finally, increase the `calcite.version` value in `/gradle.properties` and commit & push
+the change with the message "Prepare for next development iteration".
+
+Send an email to [dev@calcite.apache.org](mailto:dev@calcite.apache.org) notifying
+that `master` code freeze is over and commits can resume.
```

Comment:
Change to 'Re-open the master branch. Send an email ...'. Documentation is clearer if you lead with the purpose/high-level action.

Reply:
Changed